exercise6.rb

```diff
@@ -34,4 +34,6 @@ def print_details
 end
     
 case1=Bike.new("apache" , 58000 ,"The Bike showroom" ,12.5)
-puts case1.print_details
\ No newline at end of file
+puts case1.print_details
```

Comment:
Please read the question carefully then code.
It is clearly mentioned in the question that we have to override `to_s` method